### PR TITLE
NP-46969-no-time-limit-upload-to-sandbox-and-dev

### PR DESCRIPTION
**Pull request by anetteOlli**
Lagt til disse feltene i credentials fila.

### CODE REVIEW

File: src/main/java/no/sikt/nva/BrageMigrationCommand.java [hunk 1]
```diff
@@ -660,9 +660,11 @@ public AwsCredentialsProvider getCredentialsProviderAwsSdk() {
         public AWSCredentialsProvider getCredentialsProviderAwsAuth() {
             switch (this) {
                 case EXPERIMENTAL:
+                    return new DefaultAWSCredentialsProviderChain();
                 case SANDBOX:
+                    return getProfileCredentialsProviderAuth("nva-sandbox-brage-migration");
                 case DEVELOP:
-                    return new DefaultAWSCredentialsProviderChain();
+                    return getProfileCredentialsProviderAuth("nva-dev-brage-migration");
```
Comment (anetteOlli) [inline src/main/java/no/sikt/nva/BrageMigrationCommand.java:667]: Nice med denne metoden.

Comment (truhacevkir) [reply src/main/java/no/sikt/nva/BrageMigrationCommand.java:667]: Nice, but you should add to the the second method as well:

`public AwsCredentialsProvider getCredentialsProviderAwsSdk()`

We ended up using two different auth libraries in brage-migration, so both implementations are needed:
`software.amazon.awssdk.auth.credentials` 
`com.amazonaws.auth`

Comment (truhacevkir) [reply src/main/java/no/sikt/nva/BrageMigrationCommand.java:667]: Or you can refactor so we use only one of implementations :)

Comment (anetteOlli) [reply src/main/java/no/sikt/nva/BrageMigrationCommand.java:669]: Nice catch.

**Review by truhacevkir — APPROVED** [review]

**Pull request merged**